src/text/3-applications-subsections/3D-analysis.tex

```diff
@@ -1,12 +1,26 @@
 \subsection{3D Analysis}
 \label{ssec:3d-analysis}
-
 \todo{Laura}
+The 1D analysis approach is a powerful tool to measure the spectrum of an isolated source. However, more complicated situations, such as field of views containing several overlapping sources, require a more careful treatment. Because it is based on integration over a region, the 1D approach can't disentangle the contribution of each source to the total flux in the selected region. Similarly, sources with extended or complex morphology can result in the measured flux being underestimated, and heavily dependent on the choice of extraction region.
```

Comment:
3D analysis also has some more advantages over the 1D style (eg: taking energy dependent psf into account, for studies of very large sources covering the full FoV, etc). I guess you can lessen the comparison with the 1D case, and mention a 3D likelihood as performed, eg, in Fermi-LAT (and refer to Lars' validation paper).

Reply:
Very good points. I thought the 1D comparison would be a good way to connect this section to the previous one, but maybe it is a bit limiting... I'll rethink it and add a citation to Lars' paper. Thanks!!